src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -83,6 +83,16 @@
                (and @expanded? (< end-pan-y collapse-threshold))
                (reset! expanded? false))))))))
 
+(defn close-bottom-sheet-fn
```

Comment:
could you please elaborate on why we need this function and in the view namespace ?